package.json

```diff
@@ -28,14 +28,11 @@
   "activationEvents": [
     "onLanguage:flux",
     "onView:influxdb",
+    "onCommand:influxdb.runQuery",
     "onCommand:influxdb.addConnection",
-    "onCommand:influxdb.editConnection",
-    "onCommand:influxdb.switchConnection",
```

Comment:
Am I reading this right that switchConnection was renamed to activateConnection? And the `This` variants are removed?

Reply:
Yeah, that's it. The older, removed commands, didn't make any sense in the UI. Like, the idea was "click over here in this unrelated area and select the connection you want to switch to/delete." When @scbrickley made them accessible by right clicking the connection itself, I realized there was really no purpose in keeping the other mechanisms.

Reply:
Are users still able to switch/edit/remove connections via the command palette?

Reply:
> Are users still able to switch/edit/remove connections via the command palette?

No, as that didn't make a lot of sense.